packages/analytics/amplify_analytics_pinpoint/example/tool/pull_test_backend.sh

```diff
@@ -4,4 +4,4 @@
 
 set -e
 
-aws s3 cp s3://$AFS_ANALYTICS_BUCKET_NAME/amplifyconfiguration.dart lib/amplifyconfiguration.dart
+aws s3 cp s3://$AFS_ANALYTICS_GEN2_BUCKET_NAME/amplify_outputs.dart lib/amplify_outputs.dart 
```

Comment:
open question: do we want to run the test suite with both config versions?

Reply:
IMO we should not. The only benefit of running it with both configs is to test the config mapping. The analytics config only has 2 properties. We have unit tests to ensure those two properties are mapped correctly. No need to run the full e2e test suite for that.

I think the same argument hold true for other categories. While the mapping is a bit more complex in other categories (especially auth) I would rather invest some time into writing unit tests for the mapping rather than relying on the e2e suite.

Reply:
+1 ^^